addons/custom-block-shape/userscript.js

```diff
@@ -0,0 +1,246 @@
+export default async function ({ addon, global, console }) {
+  var multiplier = addon.settings.get("paddingSize") / 100;
```

Comment:
Its overridden on L22?

Reply:
That's a leftover from some attempt at something a while ago, so it can go back down by the other setting variables.